src/components/filter/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import Button from "components/base/button";
+
+function FilterBtnContainer() {
```

Comment:
Can we rename the folder by matching our main function here? Like from `filter` to `filterBtnContainer`